conversation_reconstruction/dataflow/write_page_states_to_spanner/spanner_write_test.py

```diff
@@ -4,21 +4,25 @@
 from __future__ import division
 from __future__ import print_function
 import unittest
-from write_utils.write import SpannerWriter 
+import os
+from write_utils.write import SpannerWriter
 from google.gax import retry
 
 
 class SpannerWriteTest(unittest.TestCase):
 
   def test_spanner_write(self):
     writer = SpannerWriter('wikiconv', 'page_info')
```

Comment:
Am I missing something, or is this test actually reading from live spanner table? Unit tests should not speak to real spanner instances. We should be using a fake spanner, or mocks I think...